cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -843,6 +859,74 @@ impl ManagedEnvironment {
         Ok(result)
     }
 
+    /// Create a new generation from local changes,
+    /// and updates the generation lock.
+    ///
+    /// If no changes exist, returns without further action.
+    ///
+    /// TODO: this should do a `build` before committing to a generation
+    pub fn sync_local(&self, flox: &Flox) -> Result<(), ManagedEnvironmentError> {
```

Comment:
Suggestion: use a more descriptive name, `sync_local` could mean pulling changes or pushing changes. Maybe `create_generation_from_local_changes` since it says exactly what it does.

Reply:
makes sense, made the name more descriptinve